os/booting-on-ec2.md

```diff
@@ -152,8 +152,7 @@ storage:
     - mount:
         device: /dev/xvdb
         format: ext4
-        create:
-          force: false
+        wipe_filesystem: true
```

Comment:
This changes meaning; is the original wrong?

Reply:
See the commit message.